### PR TITLE
Update fournisseur.facture.class.php

### PR DESCRIPTION
**Pull request by developmentOYR**
IRPF reset when editing the label of a supplier invoice #29049

### CODE REVIEW

File: htdocs/fourn/class/fournisseur.facture.class.php [hunk 1]
```diff
@@ -1243,7 +1243,7 @@ public function update($user = null, $notrigger = 0)
 		$sql .= " close_code=".(isset($this->close_code) ? "'".$this->db->escape($this->close_code)."'" : "null").",";
 		$sql .= " close_note=".(isset($this->close_note) ? "'".$this->db->escape($this->close_note)."'" : "null").",";
 		$sql .= " localtax1=".(isset($this->localtax1) ? ((float) $this->localtax1) : "null").",";
-		$sql .= " localtax2=".(isset($this->localtax2) ? ((float) $this->localtax2) : "null").",";
+		$sql .= " localtax2=".(isset($this->total_localtax2) ? ((float) $this->total_localtax2) : "null").",";
```
Comment (eldy) [inline htdocs/fourn/class/fournisseur.facture.class.php:1246]: If we use amount into this->total_locatax2 here, we must also use this->total_locatax1 at the previous line.

Comment (developmentOYR) [reply htdocs/fourn/class/fournisseur.facture.class.php:1246]: Hello Eldy, I just made the changes you asked me for. I have carried out the appropriate checks to verify that everything is working correctly.

All the best

**Review by developmentOYR — COMMENTED** [review]
Hello Eldy, I just made the changes you asked me for. I have carried out the appropriate checks to verify that everything is working correctly.

All the best

**Comment by developmentOYR** [comment]
Dear Eldy:

Could you tell me more or less what version these changes will be in?

All the best

**Comment by sonikf** [comment]
Hi @developmentOYR,
It will be included in v. 20.0.0
You can close the issue now!

**Pull request merged**